### PR TITLE
Add full author data to messages

### PR DESCRIPTION
**Pull request by jay3332**

### CODE REVIEW

File: ferrischat_webserver/src/messages/send_message.rs [hunk 1]
```diff
@@ -65,11 +65,34 @@ pub async fn create_message(
         });
     }
 
+    let author = match sql::query!("SELECT * FROM users WHERE id = $1", bigint_author_id)
```
Comment (tazz4843) [inline ferrischat_webserver/src/messages/send_message.rs:68]: sqlx

**Review by tazz4843 — CHANGES_REQUESTED** [review]
someone is going to need to `cargo check` this

**Comment by hydrostaticcog** [comment]
Just as a general rule of thumb, you should at the very minimum `cargo check` and `cargo fmt` any PR. It's be even better if you actually built and tested though.

**Review by tazz4843 — APPROVED** [review]
i'll be damned 
kuva

File: ferrischat_webserver/src/messages/get_messages.rs [hunk 2]
```diff
@@ -21,14 +21,39 @@ pub async fn get_message(req: HttpRequest, _: crate::Authorization) -> impl Resp
...
-            Some(m) => HttpResponse::Ok().json(Message {
-                id: message_id,
-                content: m.content,
-                channel_id,
-                author_id: bigdecimal_to_u128!(m.author_id),
-                edited_at: m.edited_at,
-                embeds: vec![],
-            }),
+            Some(m) => {
+                let author = match sql::query!("SELECT * FROM users WHERE id = $1", m.author_id)
```
Comment (hydrostaticcog) [inline ferrischat_webserver/src/messages/get_messages.rs:25]: `sqlx::query` not sql

**Comment by hydrostaticcog** [comment]
This needs to be reverted ASAP

**Pull request merged**